frontend/components/Links.jsx

```diff
@@ -0,0 +1,44 @@
+import Link from "next/link";
+import CategoryServicesDropdown from "./CategoryServicesDropdown";
+import styles from "../styles/navBar.module.css";
+
+export default function Links({ isOpened, categories }) {
+  return (
+    <>
+      <Link href='/'>
+        <a
+          className={`${
+            isOpened ? styles.main_nav_links_burger : styles.main_nav_links
+          }`}
+        >
+          Главная
+        </a>
+      </Link>
+      {categories.map((category) => (
+        <CategoryServicesDropdown
+          category={category}
+          key={category.title}
+          isOpened={isOpened}
+        />
+      ))}
+      <Link href='/gallery'>
+        <a
+          className={`${
+            isOpened ? styles.main_nav_links_burger : styles.main_nav_links
+          }`}
+        >
+          Галерея
+        </a>
```

Comment:
Please, don't overstyle your code. It perfectly fits in one line
```suggestion
        <a className={`${isOpened ? styles.main_nav_links_burger : styles.main_nav_links}`}>
          Галерея
        </a>
```

Reply:
This was done by the code formatter. I will trace